docs/cristin.ttl

```diff
@@ -431,11 +431,12 @@ See varbeid.kildekode. Mostly superseded by VARBEID_KILDE. It contains only the
 :sprakkode
     a               :cristinfield ;
     rdfs:comment    """
-    The language of the title and the abstract included in the VARARBEID_SPRAK entry.
+    The language of the title and the abstract included in the VARARBEID_SPRAK or the TIDSSKRIFT entry.
```

Comment:
Is it the same encoding in both domains? Also is it in the TIDSKRIFT entry is it the language code of the article or the journal? Is it possible that the article has a different language than the one appearing int the TIDSSKRIFT?

For example the entry 1055776 has a different language for the journal and a different language for the article.